core/src/com/mygdx/crazybomber/Room.java

```diff
@@ -0,0 +1,145 @@
+//TODO: when player enters, avatar is red for not ready, if blank, leave blank
+package com.mygdx.crazybomber;
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+public class Room implements Screen {
+    private Sprite background,avatar, avatarReady, avatarNotReady;
+    private SpriteBatch batch;
+    private Stage stage;
+    private Table table;
+    private TextButton buttonBack, buttonReady, buttonStart;
+    private Label heading;
+    private Skin skin;
+    private TextureAtlas atlas;
+    private Texture splashTexture;
+    private MapRepository mapRepository;
+    private StateRepository stateRepository;
+    private boolean ready = false;
+
+    public Room (Texture texture, final StateRepository stateRepository, MapRepository mapRepository) {
+        this.mapRepository = mapRepository;
+        this.splashTexture = texture;
+        this.stateRepository = stateRepository;
+    }
+
+    @Override
+    public void show() {
+        batch = new SpriteBatch();
+        background = new Sprite(splashTexture);
+        background.setSize(Gdx.graphics.getWidth(), Gdx.graphics.getHeight());
+
+        avatarReady = new Sprite(new Texture("ready.png"));
+        avatarReady.setPosition(100,300);
+        avatarReady.setSize(200,200);
+        avatarNotReady = new Sprite(new Texture("notready.png"));
+        avatarNotReady.setPosition(100,300);
+        avatarNotReady.setSize(200,200);
+        avatar = avatarNotReady;
+
+        stage = new Stage();
+        Gdx.input.setInputProcessor(stage);
+        atlas = new TextureAtlas("ui/button.pack");
+        skin = new Skin(Gdx.files.internal("menuSkin.json"),atlas);
+        table = new Table(skin);
+        table.setBounds(0,0,Gdx.graphics.getWidth(),Gdx.graphics.getHeight());
+
+        buttonBack = new TextButton("BACK", skin);
+        buttonBack.addListener(new ClickListener() {
```

Comment:
Reminder to follow a standard for all buttonBacks.

Reply:
-copied from above-
main menu > main rooms (choosing the rooms)> room (specific room)> map (specific map)
back button here goes map > main rooms directly (it doesn't go back to to the specific room)